CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # New Relic Ruby Agent Release Notes
 
+## dev
+
+- **Feature: Deprecate memcached and memcache-client instrumentation**
+
+  Instrumentation for the memcached and memcache-client libraries is deprecated and will be removed during the next major release.
```

Comment:
@fallwith  - I decided to add the deprecations here instead of making a ticket, jysk